compute/apple-silicon/how-to/connect-to-mac-mini-m1.mdx

```diff
@@ -49,7 +49,7 @@ You are now logged in and can launch applications just as if you were using your
 
 If you are using Linux and you experience problems using the [VNC button](#how-to-connect-via-vnc) to connect to your Mac mini M1, you can connect directly from a VNC client. In the example below, we use Remmina.
 
-1. [Download and install Remmina](https://remmina.org/how-to-install-remmina/), if necessary. Note that Remmina is included in most Linux distributions as standard, however.
+1. [Download and install Remmina](https://remmina.org/), if necessary. Note that Remmina is included in most Linux distributions as standard, however.
```

Comment:
I think we should remove the link because everything redirects to a 404 once you're on their website and click the pages/links

Reply:
I'll add their doc on Gitlab https://gitlab.com/Remmina/Remmina/-/wikis/home